### PR TITLE
input docs: seek +exact clarification

### PR DESCRIPTION
**Pull request by NoSuck**
I agree that my changes can be relicensed to LGPL 2.1 or later.

### CODE REVIEW

File: DOCS/man/input.rst [hunk 1]
```diff
@@ -107,8 +107,9 @@ List of Input Commands
 
     Multiple flags can be combined, e.g.: ``absolute+keyframes``.
 
-    By default, ``keyframes`` is used for relative seeks, and ``exact`` is used
-    for absolute seeks.
+    By default, ``keyframes`` is used for ``relative``, ``relative-absolute``,
```
Comment (qmega) [inline DOCS/man/input.rst]: s/relative-absolute/relative-percent/

Comment (NoSuck) [reply DOCS/man/input.rst]: 😅

**Comment by NoSuck** [comment]
Fixed.

**Comment by sfan5** [comment]
please squash your commits, github doesn't allow me to do that on merge

**Comment by NoSuck** [comment]
Done.

**Pull request merged**